.github/workflows/lineage-management.yml

```
@@ -0,0 +1,220 @@
+          echo "draft_count=$DRAFT_COUNT"
+          echo "approved_count=$APPROVED_COUNT"
+          echo "rejected_count=$REJECTED_COUNT"
+          echo "invalid_refs=$INVALID_REFS"
+        } >> "$GITHUB_OUTPUT"
+
+        echo "Validation complete: $TOTAL_LINEAGES total lineages"
+
+    - name: Cleanup old draft lineages
+      if: github.event.inputs.action == 'cleanup-drafts' || github.event_name == 'schedule'
+      id: cleanup
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:27: Double quote to prevent globbing and word splitting

(shellcheck)